records_mover/db/mysql/load_options.py

```diff
@@ -252,7 +252,10 @@ def mysql_load_options(unhandled_hints: Set[str],
     #
     # To address, we'll want to look into "set trade_date" per
     # https://stackoverflow.com/questions/44171283/load-data-local-infile-with-sqlalchemy-and-pymysql
+    # TODO update above comment
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker